nightlies/cron_wrapper

```diff
@@ -8,7 +8,9 @@ if [ -z $1 ]; then
 fi
 
 # set permanent $LOG file var
+# set $LOG_LOCKED_ERR in case locking failed
```

Comment:
actual name is LOG_LOCK_ERR.  comment should probably go above the line setting the variable instead of above the LOG= line.